src/raml.coffee

```diff
@@ -40,28 +40,46 @@ class @FileReader
   Read file from the network.
   ###
   fetchFileAsync: (file) ->
+    if window?
+      return @fetchFileAsyncBrowser file
+
+    @fetchFileAsyncNode file
+
+  fetchFileAsyncNode: (file) ->
     deferred = @q.defer()
 
-    if window?
-      xhr = new XMLHttpRequest()
-    else
-      xhr = new (require('xmlhttprequest').XMLHttpRequest)()
+    require('got').get(file, (err, data, res) ->
```

Comment:
we will not be sending the `Accept` header if the request is not in-browser?

I think that initially it was added because some server implementations (like `apikit`) required the header to be there to serve RAML. This might break that behaviour if we change it